charts/app/Chart.yaml

```diff
@@ -4,3 +4,8 @@ description: PINS Appeal Process
 type: application
 version: 0.1.0
 icon: https://github.com/planning-inspectorate.png
+dependencies:
+  - name: openfaas-amqp1-0-connector
+    alias: horizon-householder-appeal-publish
+    version: ^1.0.0
+    repository: https://helm.simonemms.com
```

Comment:
This is being put as my Helm repo for the time being. We have a task for PINS to fork this repo so it's not an external dependency